portfolio/src/main/webapp/script.js

```diff
@@ -27,32 +27,34 @@ const MILKSHAKE_IMG_CAPTION = 'Milkshakes are my favorite food/drink. I had ' +
 
 const FISH_IMG_CAPTION = 'I took AP art my senior year of high school. ' +
     'I\'ve been around photography my whole life so it\'s what I chose ' +
-    'to do for my portfolio. '
+    'to do for my portfolio. ';
 
 function showPictureAndCaption() {
-    const picturesAndCaptions = [['images/Snow.jpg', SNOW_IMG_CAPTION],
-        ['images/Chicago.jpg', CHICAGO_IMG_CAPTION],
-        ['images/Baby.jpg', BABY_IMG_CAPTION],
-        ['images/Milkshake.JPG', MILKSHAKE_IMG_CAPTION],
-        ['images/Fish.PNG', FISH_IMG_CAPTION]];
-
-    const IMG_INDEX = Math.floor(Math.random() * 5);
-
-    const IMG_ELEMENT = document.createElement('img');
-    IMG_ELEMENT.src = picturesAndCaptions[IMG_INDEX][0];
-    const IMG_CAPTION = picturesAndCaptions[IMG_INDEX][1];
-
-    const PICTURE_CONTAINER = document.getElementById('picture');
-    PICTURE_CONTAINER.innerHTML = '';
-    PICTURE_CONTAINER.appendChild(IMG_ELEMENT);
-
-    const CAPTION_CONTAINER = document.getElementById('caption');
-    CAPTION_CONTAINER.innerHTML = '';
-    CAPTION_CONTAINER.innerHTML = IMG_CAPTION;
+  const picturesAndCaptions = [
+    ['images/Snow.jpg', SNOW_IMG_CAPTION],
+    ['images/Chicago.jpg', CHICAGO_IMG_CAPTION],
+    ['images/Baby.jpg', BABY_IMG_CAPTION],
+    ['images/Milkshake.JPG', MILKSHAKE_IMG_CAPTION],
+    ['images/Fish.PNG', FISH_IMG_CAPTION],
+  ];
+
+  const IMG_INDEX = Math.floor(Math.random() * 5);
```

Comment:
These local variables, despite being const, would be better named as local variables (i.e. lowerCamelCase).  See https://google.github.io/styleguide/jsguide.html#naming-constant-names

Reply:
Done